docs/reports/requirements/4_non-func-req.md

```diff
@@ -7,3 +7,9 @@ Feature must be relatively quick. 3 minutes of audio should not take longer than
 Animation images should be of good quality: i.e high quality images or models created by a professional artist
 Animation should appear crisp. (minimum 24 frames per second.) 
 
+# PDF CONTENT 
+## Non-Functional Requirements and Environmental Constraints.
+<ul style = "list-style-type: none; margin-left: -25px;">
+    <li >The client had stated the feature should be highly portable. This is due to Kukarella’s team wishing for a modular, easy-to-integrate approach for their codebase. As such we choose to package our feature as an API. Responsiveness was also required by the client. At the time of writing this report, our team has decided that three minutes of spoken audio should not take longer than a minute for the feature to process. (This metric will be reassessed during the agile development phase.) Lastly, our client noted that the appearance of the feature should be professional and competitive. This was later clarified to mean our animations needed to be of high quality, with convincing lip synchronization and assets provided by professional graphic artists. (The client has agreed to provide funding for this.) Our team felt the animation should conform to the minimum 24-fps standard that the animation industry uses—as anything less could give a feeling of a stuttering or slow response.  </li>
```

Comment:
```suggestion
The client stated the feature should be highly portable. This is due to Kukarella’s team wishing for a modular, easy-to-integrate approach for their codebase. As such, we chose to package our feature as an API. Responsiveness was also required by the client. Our team has decided that three minutes of spoken audio should not take longer than a minute for the feature to process; although, this metric will be reassessed during the agile development phase. Lastly, our client noted that the appearance of the feature should be professional and competitive. This was later clarified to mean that our animations will need to be of high quality, with convincing lip synchronization and assets provided by professional graphic artists. It should be noted the client already has a professional artist and agreed to fund this portion of the project. Our team felt the animation should conform to the minimum 24-fps standard that the animation industry uses—as anything less could give a feeling of a stuttering or slow response.
```

Reply:
I recommend to also remove the HTML formatting and to incorporate the notes in parentheses into full sentences.